### PR TITLE
Change shell command for docker containers

### PR DESCRIPTION
**Pull request by neewy**
Previously, it had bug with lack of trailing slash

### CODE REVIEW

**Comment by l4l** [comment]
Ehm, it shouldn't be here, otherwise it consider the blank line as the part of the command

**Comment by neewy** [comment]
@l4l what is it? I do not understand

File: source/includes/_build.md [hunk 1]
```diff
@@ -81,15 +81,15 @@ Execute `run-iroha-dev.sh` again to attach to existing container.
...
-    -d redis:3.2.8
-
-docker run --name some-postgres 
-    -e POSTGRES_USER=postgres \
-    -e POSTGRES_PASSWORD=mysecretpassword \
-    -p 5432:5432 \
-    -d postgres:9.5
+docker run --name some-redis \ 
+-p 6379:6379 \
+-d redis:3.2.8 \
```
Comment (l4l) [inline source/includes/_build.md:114]: I mean this backslash

**Comment by neewy** [comment]
There was a problem with sign-off. I will close this pull request

**Pull request closed without merging**